### ISSUE TITLE
zeroth scan/map visability (pc 5.7)

### ISSUE THREAD

**Issue by StarryWisdom** [artemis-nerds/protocol-docs#163]
player 5.7 is a bitmap in the same form as first scan or second scan for map visibility (like npc 4.3) I half suggest calling this zeroth scan to make it obvious that its related in form to first scan and second scan

**Comment by NoseyNick** [comment]
Would you say this is a PlayerShip version of NPC "Map visibility (bit 4.3, int)"?
I'd vote for that name for consistency.
I should probably update my parser.pl which currently calls the NPC one "ShowOnMaps"

**Comment by StarryWisdom** [comment]
yes it is, ShowOnMaps is a good title, we are going to have ShowOnMaps appear in some of the other objects soon probably, I still slightly like ZerothScan - at a minimum we need a clear way to say ShowOnMaps/ZerothScan is the same slightly odd field as FirstScan and SecondScan (thus my suggestion of ZerothScan, a link in the docs works as well or having a single place we properly describe it or probably other)

**Comment by rjwut** [comment]
I don't feel that "zeroeth scan" is all that good of a name for it, considering that it really doesn't have anything to do with scanning. In a limited sensor range game, a ship can go out of range and become invisible, regardless of how many times it's been scanned.

**Comment by StarryWisdom** [comment]
sure - then can we have some way of linking the format of the bitmask for FirstScan SecondScan and ShowOnMap across the different objects - just so long as there is an easy way to realise all these things are highly related I'm happy

**Comment by NoseyNick** [comment]
Having renamed my PlayerShip.Unknown57 to ShowOnMaps:
```
# STAT:   54473   PlayerShip.ShowOnMaps (min 0 max 4294967295)
# STAT:    2913     PlayerShip.ShowOnMaps = 0 (0x0)
# STAT:     256     PlayerShip.ShowOnMaps = 1073741828 (0x40000004)
# STAT:     572     PlayerShip.ShowOnMaps = 1073741830 (0x40000006)
# STAT:      28     PlayerShip.ShowOnMaps = 12 (0xc)
# STAT:      21     PlayerShip.ShowOnMaps = 14 (0xe)
# STAT:       1     PlayerShip.ShowOnMaps = 20 (0x14)
# STAT:      10     PlayerShip.ShowOnMaps = 28 (0x1c)
# STAT:    4130     PlayerShip.ShowOnMaps = 4 (0x4)
# STAT:   43032     PlayerShip.ShowOnMaps = 4294967295 (0xffffffff)
# STAT:    3510     PlayerShip.ShowOnMaps = 6 (0x6)
```

**Comment by NoseyNick** [comment]
Think this was resolved by  #185 but feel free to reopen if you disagree.
